src/session.ts

```diff
@@ -175,21 +188,26 @@ class Session extends ServiceObject {
        */
       id: name,
       methods,
-      createMethod: (_, options, callback) => {
-        if (is.fn(options)) {
-          callback = options;
-          options = {};
-        }
-        return database.createSession(options, (err, session, apiResponse) => {
-          if (err) {
-            callback(err, null, apiResponse);
-            return;
-          }
+      createMethod:
+          (_: {}, optionsOrCallback: GetSession|GetSessionCallback,
```

Comment:
The name of the interface here is confusing.  If these are options for creating a new spanner session, I think `CreateSessionOptions` and `CreateSessionCallback` make more sense.

Reply:
Done